### PR TITLE
Bug#15935

### PR DESCRIPTION
**Pull request by sgrg1**
Fix issue with consecutive days range .

### CODE REVIEW

File: src/common/DatesUtilities.js [hunk 1]
```diff
@@ -104,8 +104,9 @@ const GetMaxExemptionEndDate = (monthlyData = []) => {
   const { month, year } = monthlyData.filter(hasExemption).pop();
   return endOfMonth(new Date(year, month - 1, 1));
 };
-const IsDateRangeAtLeast90Days = (fromDate, toDate) =>
-  differenceInDays(toDate, fromDate) >= 90;
+const IsDateRangeAtLeast90Days = (fromDate, toDate) => {
+  return differenceInDays(toDate, fromDate) + 1 >= 90;
```
Comment (martypowell) [inline src/common/DatesUtilities.js]: why are we adding 1 here?

Comment (sgrg1) [reply src/common/DatesUtilities.js]: If you have todate: 02/29/2020 and from date 02/01/2020. differenceInDays function is returning 28 days(29-1) but actually we wanted total 29 days. That is why I am adding +1 to compensate the missing day.

Comment (martypowell) [reply src/common/DatesUtilities.js]: can we add a comment to day that differenceInDays does not account for full days for example
jan 1 2019 to jan 31 2019 returns 30 days?

Comment (sgrg1) [reply src/common/DatesUtilities.js]: comment added

**Pull request merged**